recipe/meta.yaml

```diff
@@ -25,51 +25,31 @@ requirements:
     - wheel
   run:
     - python
-    - arrow <3.0,>=1.2.0
-    - backoff <4.0,>=2.2.1
-    - beautifulsoup4 <6.0,>=4.8.0
-    - click <10.0
-    - croniter <1.5.0,>=1.3.0
-    - dateutils <2.0
-    - deepdiff <8.0,>=5.7.0
-    - fastapi <2.0,>=0.92.0
-    - fsspec <2025.0,>=2022.5.0
-    - inquirer <5.0,>=2.10.0
-    - jinja2 <5.0
-    - lightning-cloud >=0.5.38
-    - lightning-utilities <2.0,>=0.7.0
-    - numpy <3.0,>=1.17.2
-    - packaging
-    - psutil <7.0
-    - pydantic <2.2.0,>=1.7.4
-    - python-multipart <2.0,>=0.0.5
-    - pytorch <4.0,>=1.11.0
+    - pyyaml >=5.4,<8.0
```

Comment:
As these are, to the casual eye, quite substantial changes it might be worth commenting in the recipe and/or in the PR description that these changes are consistent with `grayskull` output

Reply:
Agreed! I've added a couple clarifying comments.